### PR TITLE
cloudflare-cli: init v3.2.0

### PR DESCRIPTION
**Pull request by colemickens**
###### Motivation for this change
Add `cfcli` (cloudflare-cli)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/node-packages/node-packages-v10.nix [hunk 1]
```diff
@@ -1872,7 +1872,7 @@ let
       packageName = "strip-ansi";
       version = "3.0.1";
       src = fetchurl {
-        url = "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz";
+        url = "http://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz";
```
Comment (c0bw3b) [inline pkgs/development/node-packages/node-packages-v10.nix]: I'm not versed in Node packaging but this feels unnecessary.

File: pkgs/development/node-packages/node-packages-v6.nix [hunk 2]
```diff
@@ -1872,7 +1872,7 @@ let
       packageName = "strip-ansi";
       version = "3.0.1";
       src = fetchurl {
-        url = "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz";
+        url = "http://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz";
```
Comment (c0bw3b) [inline pkgs/development/node-packages/node-packages-v6.nix]: Same here.

**Comment by colemickens** [comment]
Thank you for the review @c0bw3b. I tend not to edit things that have been auto-generated (in this case, the `*.nix` are generated with `generate.sh` in the same directory.

But, obviously I don't think the cases you've found are good changes.

@svanderburg Can you weigh in on this? Is there a reason that these would be getting switched to `http` rather than `https`? Should I manually undo those changes and leave them as HTTPS? Is there an upstream issue for `node2nix` that covers this case?

Thanks all.

File: pkgs/development/node-packages/node-packages-v8.nix [hunk 3]
```diff
@@ -4392,7 +4401,7 @@ let
       packageName = "bl";
       version = "0.8.2";
       src = fetchurl {
-        url = "https://registry.npmjs.org/bl/-/bl-0.8.2.tgz";
+        url = "http://registry.npmjs.org/bl/-/bl-0.8.2.tgz";
```
Comment (colemickens) [inline pkgs/development/node-packages/node-packages-v8.nix]: and here

File: pkgs/development/node-packages/node-packages-v8.nix [hunk 4]
```diff
@@ -4401,7 +4410,7 @@ let
       packageName = "bl";
       version = "1.0.3";
       src = fetchurl {
-        url = "https://registry.npmjs.org/bl/-/bl-1.0.3.tgz";
+        url = "http://registry.npmjs.org/bl/-/bl-1.0.3.tgz";
```
Comment (colemickens) [inline pkgs/development/node-packages/node-packages-v8.nix]: and here... (and a number after this too.)

**Comment by colemickens** [comment]
NOTE to self: per this NixCon talk (https://youtu.be/vv4d42vbICw?t=41m7s) consider using pnpm2nix: https://github.com/adisbladis/pnpm2nix

**Comment by colemickens** [comment]
I don't use this, don't want to maintain it, it doesn't merge, etc. Closing...

**Pull request closed without merging**